src/request.rs

```diff
@@ -42,15 +42,40 @@ pub struct Process {
 }
 
 #[derive(Debug, Deserialize, Serialize)]
-pub struct Configuration {
-    pub hog_sources: Option<Vec<String>>,
-    pub hog_protocols: Option<Vec<String>>,
-    pub publish_address: Option<SocketAddr>,
-    pub ingest_address: Option<SocketAddr>,
-    pub giganto_name: Option<String>,
-    pub dump_file: Option<String>,
-    pub dump_size: Option<usize>,
+pub enum Configs {
```

Comment:
`Config` would be more appropriate, as `enum` could only be one of the `Config` kind.

Reply:
Done